src/components/Layout/SideNav/navMap.js

```diff
@@ -127,36 +127,44 @@ export const getBottomNavs = [
   }
 ];
 
-export const getDevSubNavs = [
-  {
-    title: 'Development',
-    items: [
-      { name: 'Version management', link: '#' },
-      { name: 'Audit record', link: '#' },
-      { name: 'App information', link: '#' }
-    ]
-  },
-  {
-    title: 'Operation and maintenance',
-    items: [
-      { name: 'Monitor', link: '#' },
-      { name: 'Event', link: '#' },
-      { name: 'App information', link: '#' }
-    ]
-  },
-  {
-    title: 'Customer',
-    items: [
-      { name: 'Example', link: '#' },
-      { name: 'Work list', link: '#' },
-      { name: 'News', link: '#' }
-    ]
-  },
-  {
-    title: 'Sandbox',
-    items: [{ name: 'Example', link: '#' }, { name: 'Environment', link: '#' }]
-  }
-];
+export function getDevSubNavs(appId) {
```

Comment:
Why not use arrow function ?

Reply:
If use arrow function is error, becase there has export.

Reply:
`export const getDevSubNavs = appId => {..}`